src/pages/resultsView/pathwayMapper/PathWayMapperContainer.tsx

```diff
@@ -80,7 +171,7 @@ const PathWayMapperContainer: React.FunctionComponent<IPathwayMapperContainerPro
                             urlWrapper={resultsViewPageStore.urlWrapper}
                         />
                     </MSKTab>
-                    <MSKTab
+                    {/*<MSKTab
```

Comment:
I think it's okay to delete the old code. we can always retrieve it from the git history if we really need to.